BESMod/Systems/Demand/Building/Components/SpawnHighOrderOFD.mo

```diff
@@ -0,0 +1,972 @@
+within BESMod.Systems.Demand.Building.Components;
+package SpawnHighOrderOFD
+  model GroundFloor "Spawn Groundfloor of the AixLib High Order OFD"
+    replaceable package Medium = IBPSA.Media.Air constrainedby
+      Modelica.Media.Interfaces.PartialMedium "Medium in the component"
+        annotation (choicesAllMatching=true);
+    parameter Modelica.Units.SI.Volume VZones[5] = {0,0,0,0,0} "Volume of the zones";
+    IBPSA.BoundaryConditions.WeatherData.Bus
+        weaBus "Weather data bus" annotation (Placement(transformation(extent={{-70,78},
+              {-28,122}}),         iconTransformation(extent={{-68,92},{-48,112}})));
+    Buildings.ThermalZones.EnergyPlus_9_6_0.ThermalZone zone_living1(
```

Comment:
Is Michael naming like this, as well? If not, maybe go for convention: zoneLiving or zoneLiv